### ISSUE TITLE
Question related to text processing

### ISSUE THREAD

**Issue by amalislam675** [nadavbra/protein_bert#55]
hello,
  Is there possibility to generate the embeddings representations of the the go term definitions such as "Any process that modulates a measurable attribute of any biological process, quality or function." by fine-tuning the proteinBERT model. Will the fine tuning of the proteinBERT be beneficial for this task, instead, it is trainied on protein sequences along with GO labels. Or, would I need to pre-train the proteinBERT model from scratch? For pre-training the proteinBERT model from scratch on GO term definitions, can you give some suggestions. how much time, it will take to pretrain the model from scratch?

**Comment by nadavbra** [comment]
I'm not sure I understand the task you want the model to solve.
We pre-trained the model from scratch for about a month overall, so that's how long it's supposed to take if you use a single GPU. You can see more detailed instructions in the README on the main page of this repo.
